src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,124 @@
+package duke.storage;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Scanner;
+
+import duke.exception.DukeException;
+import duke.task.Task;
+import duke.task.TaskList;
+
+/**
+ * The Storage class handles all the operations when reading or writing files from the
+ * hard disk. .
+ */
+public class Storage {
+
+    private final Path path;
+
+    /**
+     * Takes in a given filePath and saves it for reference
+     *
+     * @param filePath file path from user input
+     */
+    public Storage(String filePath) {
+        path = Paths.get(filePath);
+    }
+
+    /**
+     * Concatenates each line of the specified file if file is valid and returns
+     * it as a String value. Otherwise, a DukeException will be thrown if there are
+     * errors loading the file.
+     *
+     * @return String representation of the specified file
+     * @throws DukeException when file is not found
+     */
+    public String load() throws DukeException {
+        if (Files.exists((path))) {
+
+            // create a File for the given file path
+            File f = new File(String.valueOf(this.path));
+
+            try {
+                Scanner s = new Scanner(f);
+                String list = "";
+                while (s.hasNext()) {
+                    list += s.nextLine();
+                    list += "|";
+                }
+                return list;
+
+            } catch (FileNotFoundException e) {
+                e.printStackTrace();
+            }
+        }
+        throw new DukeException("File does not exist");
```

Comment:
You might want to put this in a separate else clause, to show that this only happens if the file does not exist.